go/Makefile

```diff
@@ -123,11 +123,12 @@ example-test:
 	mkdir -p reports
 	set -o pipefail; \
 	../utils/cluster_manager.py start -p 6379 -r 0
-	../utils/cluster_manager.py start --cluster-mode -p 7001 7002 7003 7004 7005 7006
+	@OUTPUT=$$(python3 ../utils/cluster_manager.py start --cluster-mode); \
+	GLIDE_CLUSTER_NODES=$$(echo "$$OUTPUT" | grep 'CLUSTER_NODES=' | cut -d'=' -f2); \
 	LD_LIBRARY_PATH=$(shell find . -name libglide_rs.a|grep -w release|tail -1|xargs dirname|xargs readlink -f):${LD_LIBRARY_PATH} \
-    go test -v ./... -skip Test $(if $(test-filter), -run $(test-filter)) \
+	go test -v ./api -skip Test $(if $(test-filter), -run $(test-filter)) -clusternodes $$GLIDE_CLUSTER_NODES \
 	| tee >(go tool test2json -t -p github.com/valkey-io/valkey-glide/go/api \
-    | go-test-report -o reports/example-tests.html -t example-test > /dev/null)
+	| go-test-report -o reports/example-tests.html -t example-test > /dev/null)
```

Comment:
Failure of example test isn't propageted, `make` still exits with 0, please adjust the script.
Please also update CI to run this test

Reply:
fixed